Shock/Classes/NIO/MockNIOHTTPHandler.swift

```diff
@@ -65,16 +71,19 @@ class MockNIOHTTPHandler {
             queryParams = queryItems.reduce(into: [(String, String)](), { $0.append(($1.name, $1.value ?? "")) })
         }
         
-        return MockNIOHTTPRequest(path: path,
-                              queryParams: queryParams,
-                              method: method,
-                              headers: headers,
-                              body: body,
-                              address: address,
-                              params: params)
+        return MockNIOHTTPRequest(eventLoop: eventLoop,
+                                  path: path,
+                                  queryParams: queryParams,
+                                  method: method,
+                                  headers: headers,
+                                  body: body,
+                                  address: address,
+                                  params: params)
     }
     
-    private func handleResponse(forResponseContext middlewareContext: MiddlewareContext, in     channelHandlerContext: ChannelHandlerContext) {
+    private func handleResponse(forResponseContext middlewareContext: MiddlewareContext,
+                                in channelHandlerContext: ChannelHandlerContext,
+                                version: HTTPVersion) {
         
         // TODO
```

Comment:
It's not super clear what is To be done here? Should this comment be updated or even removed?

Reply:
Just formatting.

Reply:
👍